test/test_helper.rb

```diff
@@ -11,7 +11,7 @@
 
 VCR.configure do |config|
   config.cassette_library_dir = "test/fixtures/vcr_cassettes"
-  config.hook_into :webmock
+  config.hook_into :faraday
```

Comment:
What was this change for? (I mean it looks somewhat self explanatory, not sure why it wasn't needed before)